redaxo/src/addons/media_manager/lang/de_de.lang

```diff
@@ -166,3 +166,8 @@ media_manager_effect_contrast_notice = [-100 bis 100]. Leer lassen für Standard
 media_manager_effect_sepia = Bild: Sepia
 
 media_manager_effect_greyscale = Bild: Graustufen
+
+media_manager_effect_img2img = Bild: Bild in anderes Format konvertieren
+media_manager_effect_img2img_convertto = Zielformat
+media_manager_effect_img2img_quality = Qualität 
+media_manager_effect_img2img_quality_notice = jpg/webp: [0-100] Leer lassen für Standardwert.<br>png: [0-9]. Werte über 6-7 verlangsamen die Bilderzeugung massiv. Leer lassen für Standardwert.
```

Comment:
Wir haben eigentlich schon einen separaten Effekt, um die Default-Quality-Werte in einem MM-Typ ändern zu können: https://github.com/redaxo/redaxo/blob/master/redaxo/src/addons/media_manager/lib/effects/effect_image_properties.php

Macht es dann wirklich Sinn, hier noch eine dritte Stelle zu haben, wo man die Werte eingeben kann? Sollte man dann nicht eher den vorhandenen Effekt eben vorschalten, wenn man abweichende Werte von den Default-Werten nutzen möchte?

Reply:
> Macht es dann wirklich Sinn, hier noch eine dritte Stelle zu haben, wo man die Werte eingeben kann? Sollte man dann nicht eher den vorhandenen Effekt eben vorschalten, wenn man abweichende Werte von den Default-Werten nutzen möchte?

Oder man macht einen Neuanfang und kombiniert das ganze in diesem neuen Effekt und lässt den ausreifen. Die alten Effekte stammen noch aus einer Zeit vor Responsive images, srcset, picture, CSS3 und Co. - die ganzen Effekte wie Schatten, Kontur, Spiegelung, etc. braucht doch kein Mensch mehr, oder? Dafür fehlen Autokorrektur und Co.

Reply:
Ich würde gerne den neuen Effekt passend in die vorhandene Struktur integrieren, und jetzt keine grundsätzlichen Richtungsänderungen hier im PR einschlagen.

Reply:
Also ich finde es praktischer wenn die Einstellungen direkt hier beim Effekt sind.
Ich denke beim Effekt image_properties ist für viele auch nicht ersichtlich was der eigentlich macht :)

Reply:
Und man hat bei mehreren Profilen wieder weniger Effekte die man hinzufügen muss :) @alxndr-w

Reply:
> Und man hat bei mehreren Profilen wieder weniger Effekte die man hinzufügen muss :) @alxndr-w

Das verstehe ich nicht. Ich sehe da keine Ersparnis. Aber passt schon, wichtiger ist mir, dass das erstmal drin ist.

Reply:
Also ich finde es nicht gut, wenn wir hier im Effekt eine dritte Stelle haben. Denke eigentlich auch nicht, dass es die Regel sein wird, dass man hier von den Standardwerten aus den Einstellungen abweichen möchte.
Und wenn doch, dann haben wir ja nun mal schon den anderen Effekt. Der aktuell hier übrigens komplett ignoriert werden würde, da aktuell default die globalen Einstellungen genommen werden. Ließe sich ändern, aber mir leuchtet halt noch nicht ein, weswegen wir hier eine dritte Stelle benötigen.

Reply:
Ja, so sollte es passen.